simulator/func_sim/func_sim.cpp

```diff
@@ -64,15 +64,19 @@ void FuncSim<ISA>::init( const std::string& tr)
 }
 
 template <typename ISA>
-void FuncSim<ISA>::run( const std::string& tr, uint64 instrs_to_run)
+Simulator::RunResult FuncSim<ISA>::run( const std::string& tr, uint64 instrs_to_run)
```

Comment:
Maybe, it's worth to return FuncInstr here....

Reply:
At this moment, it would be hard to return FuncInstr from PerfSim. I think we may change it later, when traps would be implemented in PerfSim.